isaacgym_utils/assets/franka.py

```diff
@@ -1,6 +1,11 @@
 import numpy as np
 from pathlib import Path
 
+# needed for IK
+import roboticstoolbox as rtb
+from spatialmath import SE3, SO3
```

Comment:
remove `spatialmath`

Reply:
we need to keep `spatialmath` per https://github.com/iamlab-cmu/isaacgym-utils/pull/29#discussion_r789678408